CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-cmake_minimum_required(VERSION 3.1.0)
-project(PortAudio VERSION 19.8)
+cmake_minimum_required(VERSION 3.10)
+project(PortAudio VERSION 19.8 LANGUAGES C CXX)
```

Comment:
`LANGUAGES` is optional and C and CXX are enabled by default by CMake `By default C and CXX are enabled if no language options are given.` (https://cmake.org/cmake/help/latest/command/project.html), therefore specifying them explicitly is not needed.

Reply:
Just for C the change made sense. Now not more. Thanks for pointing this out, I dropped the second commit.